libs/pages/settings/src/lib/ui/page-organization-billing-summary/show-usage-value-modal/show-usage-value-modal.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { getByDisplayValue, getByTestId, render } from '__tests__/utils/setup-jest'
```

Comment:
Same here, it's better to use our new lib for testing `@qovery/shared/util-tests`